libnavigation-core/src/test/java/com/mapbox/navigation/core/reroute/MapboxRerouteControllerTest.kt

```diff
@@ -169,6 +178,55 @@ class MapboxRerouteControllerTest {
         }
     }
 
+    @Test
+    fun reroute_success_from_alternative() {
+        mockkStatic(
+            "com.mapbox.navigation.base.internal.utils.NavigationRouteEx",
+            "com.mapbox.navigation.base.internal.utils.RouteProgressEx"
+        ) {
```

Comment:
nice ❤️ 
I like this per method setup more that per class using `@Before` and `@After`